From 742fcbd0cf879d0be039368b11a0b1893c430030 Mon Sep 17 00:00:00 2001
From: Elaine Zhang <zhangqing@rock-chips.com>
Date: Mon, 10 Oct 2016 17:07:48 +0800
Subject: [PATCH] ARM64: dts: rk3399: add regulator-ramp-delay for dcdc

used to calculate the delay time for change dcdc voltage.

Change-Id: I6bb462ef087b9ce6aa98991a1b961ed5f57bb3c8
Signed-off-by: Elaine Zhang <zhangqing@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-box-808-android.dts | 2 ++
 arch/arm64/boot/dts/rockchip/rk3399-box.dtsi            | 2 ++
 arch/arm64/boot/dts/rockchip/rk3399-evb-rev1.dtsi       | 3 +++
 arch/arm64/boot/dts/rockchip/rk3399-evb-rev2.dtsi       | 3 +++
 arch/arm64/boot/dts/rockchip/rk3399-evb-rev3.dtsi       | 2 ++
 arch/arm64/boot/dts/rockchip/rk3399-sapphire.dtsi       | 2 ++
 6 files changed, 14 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-box-808-android.dts b/arch/arm64/boot/dts/rockchip/rk3399-box-808-android.dts
index 2c35054ef6b2..3f083c50c131 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-box-808-android.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-box-808-android.dts
@@ -349,6 +349,7 @@
 				regulator-name = "vdd_log";
 				regulator-min-microvolt = <750000>;
 				regulator-max-microvolt = <1350000>;
+				regulator-ramp-delay = <6001>;
 				regulator-always-on;
 				regulator-boot-on;
 				regulator-state-mem {
@@ -361,6 +362,7 @@
 				regulator-name = "vdd_cpu_l";
 				regulator-min-microvolt = <750000>;
 				regulator-max-microvolt = <1350000>;
+				regulator-ramp-delay = <6001>;
 				regulator-always-on;
 				regulator-boot-on;
 				regulator-state-mem {
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-box.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-box.dtsi
index 0d77c8192b61..54528dde37e9 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-box.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-box.dtsi
@@ -523,6 +523,7 @@
 				regulator-name = "vdd_center";
 				regulator-min-microvolt = <750000>;
 				regulator-max-microvolt = <1350000>;
+				regulator-ramp-delay = <6001>;
 				regulator-always-on;
 				regulator-boot-on;
 				regulator-state-mem {
@@ -534,6 +535,7 @@
 				regulator-name = "vdd_cpu_l";
 				regulator-min-microvolt = <750000>;
 				regulator-max-microvolt = <1350000>;
+				regulator-ramp-delay = <6001>;
 				regulator-always-on;
 				regulator-boot-on;
 				regulator-state-mem {
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev1.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev1.dtsi
index 249d4d7ac811..767edf195cb2 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev1.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev1.dtsi
@@ -126,6 +126,7 @@
 				regulator-name = "vdd_gpu";
 				regulator-min-microvolt = <712500>;
 				regulator-max-microvolt = <1500000>;
+				regulator-ramp-delay = <8000>;
 				regulator-always-on;
 				regulator-boot-on;
 			};
@@ -163,6 +164,7 @@
 				regulator-boot-on;
 				regulator-min-microvolt = <750000>;
 				regulator-max-microvolt = <1350000>;
+				regulator-ramp-delay = <6001>;
 				regulator-name = "vdd_cpu_b";
 				regulator-state-mem {
 					regulator-off-in-suspend;
@@ -174,6 +176,7 @@
 				regulator-boot-on;
 				regulator-min-microvolt = <750000>;
 				regulator-max-microvolt = <1350000>;
+				regulator-ramp-delay = <6001>;
 				regulator-name = "vdd_cpu_l";
 				regulator-state-mem {
 					regulator-off-in-suspend;
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev2.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev2.dtsi
index b64a126a11e2..c0a4f71452fc 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev2.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev2.dtsi
@@ -287,6 +287,7 @@
 				regulator-name = "vdd_gpu";
 				regulator-min-microvolt = <735000>;
 				regulator-max-microvolt = <1400000>;
+				regulator-ramp-delay = <6000>;
 				regulator-always-on;
 				regulator-boot-on;
 			};
@@ -324,6 +325,7 @@
 				regulator-boot-on;
 				regulator-min-microvolt = <750000>;
 				regulator-max-microvolt = <1350000>;
+				regulator-ramp-delay = <6001>;
 				regulator-name = "vdd_log";
 				regulator-state-mem {
 					regulator-on-in-suspend;
@@ -336,6 +338,7 @@
 				regulator-boot-on;
 				regulator-min-microvolt = <750000>;
 				regulator-max-microvolt = <1350000>;
+				regulator-ramp-delay = <6001>;
 				regulator-name = "vdd_cpu_l";
 				regulator-state-mem {
 					regulator-off-in-suspend;
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3.dtsi
index 0714ea197c52..36cfd18a0fdc 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3.dtsi
@@ -320,6 +320,7 @@
 				regulator-boot-on;
 				regulator-min-microvolt = <750000>;
 				regulator-max-microvolt = <1350000>;
+				regulator-ramp-delay = <6001>;
 				regulator-name = "vdd_log";
 				regulator-state-mem {
 					regulator-on-in-suspend;
@@ -332,6 +333,7 @@
 				regulator-boot-on;
 				regulator-min-microvolt = <750000>;
 				regulator-max-microvolt = <1350000>;
+				regulator-ramp-delay = <6001>;
 				regulator-name = "vdd_cpu_l";
 				regulator-state-mem {
 					regulator-off-in-suspend;
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-sapphire.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-sapphire.dtsi
index 7104557e2e8e..7785e94ef81f 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-sapphire.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-sapphire.dtsi
@@ -320,6 +320,7 @@
 				regulator-boot-on;
 				regulator-min-microvolt = <750000>;
 				regulator-max-microvolt = <1350000>;
+				regulator-ramp-delay = <6001>;
 				regulator-name = "vdd_center";
 				regulator-state-mem {
 					regulator-off-in-suspend;
@@ -331,6 +332,7 @@
 				regulator-boot-on;
 				regulator-min-microvolt = <750000>;
 				regulator-max-microvolt = <1350000>;
+				regulator-ramp-delay = <6001>;
 				regulator-name = "vdd_cpu_l";
 				regulator-state-mem {
 					regulator-off-in-suspend;
-- 
2.35.3

